frontend/app/adapters/def-image-upload.js

```diff
@@ -2,9 +2,9 @@ import DS from 'ember-data';
 import ENV from '../config/environment';
 
 const { APP } = ENV;
-const { RESTAdapter } = DS;
+const { JSONAPIAdapter } = DS;
```

Comment:
Why this change?

Reply:
This is because the backend api needs the data to be complying with JSON spec an not REST spec, as ember data treats them both differently.

Reply:
Oh! All right! 👍